chainer/functions/connection/n_step_lstm.py

```diff
@@ -194,49 +237,100 @@ def n_step_bilstm(
     Args:
         n_layers(int): Number of layers.
         dropout_ratio(float): Dropout ratio.
-        hx (chainer.Variable): Variable holding stacked hidden states.
-            Its shape is ``(S, B, N)`` where ``S`` is number of layers and is
+        hx (~chainer.Variable): Variable holding stacked hidden states.
+            Its shape is ``(2S, B, N)`` where ``S`` is number of layers and is
             equal to ``n_layers``, ``B`` is mini-batch size, and ``N`` is
-            dimention of hidden units.
-        cx (chainer.Variable): Variable holding stacked cell states.
+            dimension of hidden units. Because of bi-diraction, the first
+            dimension length is ``2S``.
+        cx (~chainer.Variable): Variable holding stacked cell states.
             It has the same shape as ``hx``.
-        ws (list of list of chainer.Variable): Weight matrices. ``ws[i]``
-            represents weights for i-th layer.
-            Each ``ws[i]`` is a list containing eight matrices.
-            ``ws[i][j]`` is corresponding with ``W_j`` in the equation.
-            Only ``ws[0][j]`` where ``0 <= j < 4`` is ``(I, N)`` shape as they
-            are multiplied with input variables. All other matrices has
-            ``(N, N)`` shape.
-        bs (list of list of chainer.Variable): Bias vectors. ``bs[i]``
-            represnents biases for i-th layer.
+        ws (list of list of :class:`~chainer.Variable`): Weight matrices.
+            ``ws[2 * l + m]`` represents weights for l-th layer of m-th
+            direction. (``m == 0`` means the forward direction and ``m == 1``
+            means the backward direction.) Each ``ws[i]`` is a list containing
+            eight matrices. ``ws[i][j]`` is corresponding with :math:`W_j` in
+            the equation. ``ws[0][j]`` and ``ws[1][j]`` where ``0 <= j < 4`` is
```

Comment:
... is (I, N) shape -> ... are (I, N)-shaped

Reply:
I fixed next commit.